app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -1,5 +1,12 @@
 <ng-form name="persistentVolumeClaimForm">
   <fieldset ng-disabled="claimDisabled">
+    <div ng-if="persistentVolumeClaimForm.capacity.$error.outOfStorage" class="has-error">
+      <div class="alert alert-danger">
+        <span class="pficon pficon-error-circle-o" aria-hidden="true"></span>
+        <strong>Quota limit has been reached. New storage cannot be created. To increase your quota, contact your administrator.</strong>
```

Comment:
same comment as above, make this "Storage quota limit has been reached. You will not be able to create any new storage."